scripts/lib/CIME/case_run.py

```diff
@@ -122,6 +123,7 @@ def _run_model_impl(case, lid):
                         orig_cont = case.get_value("CONTINUE_RUN")
                         if not orig_cont:
                             case.set_value("CONTINUE_RUN", True)
+                        if not skip_pnl:
```

Comment:
This still isn't right. You've changed the original intent which was to re-run create_namelists if CONTINUE_RUN was False. For now, let's not worry about skip_pnl in this code. This code is for recovery from node failures which is a rare corner case and we don't need to worry about how it performs.